<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Transfer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../python_excel_project/code/love_icon.ico</normaloff>../python_excel_project/code/love_icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(17, 17, 17);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <item row="2" column="0">
         <widget class="QLabel" name="txtDisplaySrcPath">
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color:rgb(68,76,86);
	font: 9pt &quot;Small Fonts&quot;;
	border-style:none;
	border-radius: 10px;
	color:#FFF;
	qproperty-alignment: AlignCenter;
}</string>
          </property>
          <property name="text">
           <string>Src Dir Path Display</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="btnCopy">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(0, 170, 0);
	font: 9pt &quot;Small Fonts&quot;;
border:none;
border-radius:10px;
color:#FFF
}</string>
          </property>
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="txtDisplayDstPath">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color:rgb(68,76,86);
	font: 9pt &quot;Small Fonts&quot;;
	border-style:none;
	border-radius: 10px;
	color:#FFF;
	qproperty-alignment: AlignCenter;
}</string>
          </property>
          <property name="text">
           <string>Dst Dir Path Display</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="txtDisplayFile">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color:rgb(68,76,86);
	font: 9pt &quot;Small Fonts&quot;;
	border-style:none;
	border-radius: 10px;
	color:#FFF;
	qproperty-alignment: AlignCenter;
}</string>
          </property>
          <property name="text">
           <string>Copy File Name Display</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnSrcDir">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(0, 170, 0);
	font: 9pt &quot;Small Fonts&quot;;
border:none;
border-radius:10px;
color:#FFF
}</string>
          </property>
          <property name="text">
           <string>Src Directory</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnDstDir">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(0, 170, 0);
	font: 9pt &quot;Small Fonts&quot;;
border:none;
border-radius:10px;
color:#FFF
}</string>
          </property>
          <property name="text">
           <string>Dst Directory</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QProgressBar" name="progressBar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QProgressBar{
	background-color:rgb(68,76,86);
	color:#FFF;
	border-style:none;
	border-radius: 10px;
	text-align:center
}
QProgressBar::chunk{
	background-color:rgb(255,0,127);
	margin:10px;
	border-radius:10px
}</string>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
